packages/extension-ui/src/Popup/Signing/Extrinsic/Extrinsic.stories.tsx

```diff
@@ -0,0 +1,211 @@
+      nonce: '0x00000009',
+      signedExtensions: [
+        'CheckNonZeroSender',
+        'CheckSpecVersion',
+        'CheckTxVersion',
+        'CheckGenesis',
+        'CheckMortality',
+        'CheckNonce',
+        'CheckWeight',
+        'ChargeTransactionPayment',
+      ],
+      tip: '0x00000000000000000000000000000000',
+      version: 4,
+    }
+  },
+};
+
+export const TheButtonSwapBatchTransaction: Story = {
+  args: {
+    requestPayload: {
+      specVersion: '0x00000040',
+      transactionVersion: '0x0000000e',
+      address: '5EkCk5SMQxUQGrULfDjsMeMCbcKoMgTd9ZBSrvMxiSDZUS86',
+      blockHash: '0x2b51dff9541913e577a1820e1ff20284d351c886fa0ce37ccc6b25caeb9ad3c4',
+      blockNumber: '0x01fa69f3',
+      era: '0x381f',
+      genesisHash: '0x05d5279c52c484cc80396535a316add7d47b1c5b9e0398dd1f584149341460c5',
+      method:
+        '0x0f02081206004755abb4d05d2a41cdaa7b7535a3c85b4725d26fee2a9951aeb95674c29bfb9c0003000032e90200080000d0b20f1bbd09526e9aab507309a0a3eb3d8cc469fbf27dcfcb6b64c584e968099027447192000064a7b3b6e00d000000000000000012060009526e9aab507309a0a3eb3d8cc469fbf27dcfcb6b64c584e9680990274471920007002de5c5040bff4f39278c0400910111004fa64755abb4d05d2a41cdaa7b7535a3c85b4725d26fee2a9951aeb95674c29bfb9cccd12db4c1496aae7a33d77f1a6ec7071c0dfc2a357f3db816d5e581018a199c0010a5d4e8000000000000000000000000e40b54020000000000000000000000',
+      nonce: '0x0000000a',
+      signedExtensions: [
+        'CheckNonZeroSender',
+        'CheckSpecVersion',
+        'CheckTxVersion',
+        'CheckGenesis',
+        'CheckMortality',
+        'CheckNonce',
+        'CheckWeight',
+        'ChargeTransactionPayment',
+      ],
+      tip: '0x00000000000000000000000000000000',
+      version: 4,
+    }
+  },
+};
+
+export const StakingDashboardNomination: Story = {
+  args: {
+    requestPayload: {
+      'specVersion': '0x00000040',
+      'transactionVersion': '0x0000000e',
+      'address': '5EkCk5SMQxUQGrULfDjsMeMCbcKoMgTd9ZBSrvMxiSDZUS86',
+      'blockHash': '0xce199fc12952396bff8e3c918521ea4e9ef22cadb038891893b8913477891428',
+      'blockNumber': '0x0215502c',
+      'era': '0xc802',
+      'genesisHash': '0x05d5279c52c484cc80396535a316add7d47b1c5b9e0398dd1f584149341460c5',
+      'method': '0x0f000c0800007698471eccdaf1b0374546c6666a2014d641e6c8d6f60b4c24738aa09a65882d0b00407a10f35a0108050400ec8df85d7896bcbb81d907c9f411f641fe983f5b217ab8116227eda760a6f2580808007498c40a78ff0c474c14c704ecdc88e91e68edd3fee9f65b45ec54151b237763',
+      'nonce': '0x0000001b',
+      'signedExtensions': [
+        'CheckNonZeroSender',
+        'CheckSpecVersion',
+        'CheckTxVersion',
+        'CheckGenesis',
+        'CheckMortality',
+        'CheckNonce',
+        'CheckWeight',
+        'ChargeTransactionPayment'
+      ],
+      'tip': '0x00000000000000000000000000000000',
+      'version': 4
+    }
+  },
+};
+
+export const MissingChainMetadata: Story = {
```

Comment:
Yeah, this is just the centered-by-default style for SB components and the details components is short enough in this particular story that it's centered. So that's expected.